### ISSUE TITLE
Stuck on Queuing with Strava and Garmin

### ISSUE THREAD

**Issue by catsarecool999** [cpfair/tapiriik#412]
I disconnected and reconnected. It stays on Queuing for hours and never does anything.

**Comment by alanbill99** [comment]
Yep - ditto. Between Strava (source) and Runkeeper (as I want to export gpx from there). Are the servers down?

**Comment by catsarecool999** [comment]
It just came through this morning. No calories in Garmin Connect, though! GRRRR

**Comment by Linguisht** [comment]
Me too, stuck in queuing

**Comment by Ballou** [comment]
Me too. No sync from GC since Mar 22. Stuck on queuing. Tried disconnecting my two services that I sync, GC and Dropbox. Still no joy. Now Tapiriik UI won't let me choose subfolders of DropBox/Apps either. DB Apps folder appears empty, though there are 7 subfolders including Tapiriik.

**Comment by mjclemente** [comment]
Ditto, unfortunately.

**Comment by Antash** [comment]
duplicate #416

**Comment by Ballou** [comment]
Any word on this @cpfair ? It's been a while… (no sync since 3/22 for me)
- Garmin Connect Sync queue stuck
- Tapiriik/DropBox Setup UI (w/ full authorization) does NOT show Apps/ subfolders. Only first level and default Tapiriik folder

Periodically disconnect and reconnect GC from Tapiriik, in hopes it will refresh the link. No joy. 

Paid user. My sole use of Tapiriik is to sync GC with DropBox (/Apps/iSmoothRun) for import and distribution with iOS app iSmoothRun.

**Comment by Linguisht** [comment]
I bypassed it by setting up direct sync between GC and Strava. Then the tapiirik sync to dropbox etc kicks in from strava instead.

**Comment by Ballou** [comment]
That's a good idea @Linguisht. I was just trying to limit the amount of steps to edit the details of the activity before distributing it. As it was, all I had to do was edit the GC activity (notes, title, type, etc) right after and then it ended up in DB correct and got sent out through iSR that way. I'll have to do that in GC & Strava if I want the data to have parity. 

Honestly, I'm tired of managing all this crap. Originally I had friends who couldn't decide on what platform to use, so I uploaded to a lot of them. But other than liking the ability to upload a FaceBook fitness action, I'm not really tied to any of it anymore. Most everyone is on Strava, and all the other sites seem to have direct sync connections available.

Thanks again.

**Comment by Ballou** [comment]
Well, sync partially resumed for me at about 730PM PST yesterday. Noticed some files got transferred to DB from GC via Tapiriik, but only through the end of March. April activities = no joy.

In addition, I sadly still can't get Tapiriik to show the subfolders of Apps yet where I direct my files. So I had to move them manually. For my usual workflow I need to be able to drill down sync to DB/Apps/iSmoothRun/Import. Posted a separate issue ticket about this.

Played around with direct sync of GC to Strava, but it moves EVERY activity to Strava. I like just posting my Running. I don't need everyone on Strava to know about all my resistance training sessions (which I also record on my Garmin), especially because they sync before I get a chance to edit their details. So in that form, they don't mean anything really, and look like junk posts on Strava.

**Comment by Ballou** [comment]
My GC to DropBox sync is back up. Closed #429. Not sure what the issue was. @cpfair was quick to directly address a DropBox folder issue (#428) and resolve it, but there was no communication regarding not pulling activities from GC over the last couple/three weeks. It's working for me though… Hope it's back up for all of you.

**Comment by Ballou** [comment]
Grrr. Sync is down again since April 13th. No joy pulling from GC. There was system maintenance at Garmin over the weekend. So I'm thinking that broke something? Crapsicle.
